projects/angular/src/layout/nav/aria-current-link.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive, HostBinding, OnDestroy, OnInit } from '@angular/core';
+import { RouterLinkActive } from '@angular/router';
+import { Subscription } from 'rxjs';
+
+@Directive({ selector: '[clr-aria-current-link]', host: { '[attr.aria-current]': 'stepAriaCurrent' } })
```

Comment:
We usually use camelCase for directive selectors and kebab-case for component selectors. There are few cases of kebab-case directives as they are there to overload existing components with additional functionalities. 

Also, it doesn't need to be in a square bracket.

Reply:
Updated. do you mean it doesn't need the square bracket here in the selector or when I use it.

I noticed that when I use it without the brackets, it shows up on the dom element when viewing the source as `clrariacurrentlink` and I wasn't sure if that's behavior we want/expect.

Reply:
Given that this is an attribute directive, I think the square brackets are warranted.

Reply:
Sorry for being unclear on brackets. You want to keep the brackets in the selector, but you don't need it when you use it.